docker/compose_files/docker-compose.ssl.yml

```diff
@@ -3,9 +3,8 @@ version: '3.5'
 services:
   nginx:
     ports:
-      - 443:443
+     - ${CODALAB_HTTPS_PORT}:${CODALAB_HTTPS_PORT}
```

Comment:
Where is this environment variable set?

Reply:
in `codalab_service.py` as a default (search for 443).